web_src/css/admin.css

```
@@ -15,32 +15,27 @@
   flex-wrap: wrap;
 }
 
-.admin dl.admin-dl-horizontal dd {
-  margin-left: auto;
-  width: calc(100% - 245px);
-}
-
-@media (max-width: 767.98px) {
-  .admin dl.admin-dl-horizontal dd {
-    width: calc(100% - 185px);
-  }
-}
-
-/* divider needs explicit width to become visible here */
+/* divider needs explicit width because the parent container is "display: flex" */
 .admin dl.admin-dl-horizontal .ui.divider {
   width: 100%;
 }
 
+.admin dl.admin-dl-horizontal dt,
+.admin dl.admin-dl-horizontal dd {
+  line-height: 1em;
```

```suggestion
  line-height: 1;
```

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height):

> The used value is this unitless [<number>](https://developer.mozilla.org/en-US/docs/Web/CSS/number) multiplied by the element's own font size. The computed value is the same as the specified <number>. In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance.